pkgs/os-specific/linux/kernel/cpu-cgroup-v2-patches/4.9.patch

```diff
@@ -428,7 +428,7 @@ index 000000000000..1ed7032d4472
 +solution, and details the disagreements and arguments.  The latest
 +version of this document can be found at the following URL.
 +
-+ https://git.kernel.org/cgit/linux/kernel/git/tj/cgroup.git/tree/Documentation/cgroup-v2-cpu.txt?h=cgroup-v2-cpu
```

Comment:
is patching patches necessary here? the links do still work after all (except for a few that redirect to the new urls but end up broken because they don't include revision info and the file moved since then)

Reply:
I think it's always better to update URLs when they start redirecting.  Redirects don't last forever.

Reply:
on one hand, yes. on the other this is a part of the patch content and could affect other patches added down the line. (this is admittedly extremely unlikely, not least because 4.9 will be dead in six months. not going to hold on to the point of not changing these.)